### PR TITLE
Minor usability fixes

### PR DESCRIPTION
**Pull request by hugopeixoto**
*  Make fontawesome work without javascript

When browsers don't have javascript enabled, social media links don't show up.
Fontawesome supports defaulting to css.

* Improve navigation visibility when scrolling

When scrolling, the navigation header was fixed to the top of the viewport. Due to how it's positioned, it goes behind some images.

Additionally, the background color was set to transparent. This means that the navigation text becomes unreadable when it's on top of other text.

By changing the z-index, setting background-color to white, and adding some padding, the navigation header becomes a bit more usable.

### CODE REVIEW

**Comment by silva95gustavo** [comment]
Fantastic! We really needed this.

File: _sass/minima/_layout.scss [hunk 1]
```diff
@@ -5,6 +5,9 @@
   font-size: $small-font-size;
   text-align: center;
   padding-top: 3 * $spacing-unit;
+  padding-bottom: $spacing-unit;
+  z-index: 1;
+  background-color: white;
```
Comment (mariajplmiranda) [inline _sass/minima/_layout.scss]: `background-color: $white;` 🥇

**Review by mariajplmiranda — APPROVED** [review]

File: _sass/minima/_layout.scss (hunk 1 above)
Comment (hugopeixoto) [reply _sass/minima/_layout.scss]: I am assuming you're saying I should change `white` to `$white`. will do :+1:

Comment (hugopeixoto) [reply _sass/minima/_layout.scss]: Done. Amended the existing commit to fix this.

**Pull request merged**